src/scripts/utils.py

```diff
@@ -399,3 +409,26 @@ def jinja_sparql(
         df = pd.DataFrame()
 
     return df
+
+
+def get_labels(
```

Comment:
@matentzn ORDO and OMIM had some cases with no labels.

## Sub-task list
- [ ] 1. ORDO missing labels
- [ ] 2. OMIM missing labels
- [ ] 3. Also double check if any missing labels for other ontologies

## Sub-task details
### 1. ORDO missing labels
I'm kind of baffled here. In every case I checked so far, these terms have simple class declarations, are not deprecated, and do have an `rdfs:label`. I think what I'll try is to query these by alone and see the result. Need to look at / think about it more.

Here's the full list (n=541): [ordo labels not being fetched.txt](https://github.com/monarch-initiative/mondo-ingest/files/10495344/ordo.labels.not.being.fetched.txt)

### 2. OMIM missing labels
From what I can see, it's because these are all deprecated. I will filter out these deprecated terms in my next update. 

However, @matentzn I am wondering, is it expected that the `mirror_signature` files contain deprecated terms? That is what I am using as the starting point for collecting terms. The logic I'm using for determining mappable terms that are still unmapped is: _(terms in the `mirror_signature`) - (excluded terms) - (already mapped terms)_.

Reply:
Solved. In some cases, it was because `deprecated`. In others, it was because I was querying from the mirrors.